src/Symfony/Component/Uid/Tests/Command/InspectUuidCommandTest.php

```diff
@@ -0,0 +1,216 @@
+  Canonical (RFC 4122)   d108a1a0-957e-3c77-b110-d3f912374439  
+  Base 58                Sp7q16VVeC7zPsMPVEToq2                
+  Base 32                6H12GT15BY7HVV246KZ493EH1S            
+ ---------------------- -------------------------------------- 
+
+
+EOF
+            , $commandTester->getDisplay(true));
+    }
+
+    public function testV4()
+    {
+        $commandTester = new CommandTester(new InspectUuidCommand());
+
+        $this->assertSame(0, $commandTester->execute(['uuid' => '705c6eab-a535-4f49-bd51-436d0e81206a']));
+        $this->assertSame(<<<EOF
+ ---------------------- -------------------------------------- 
+  Label                  Value                                 
+ ---------------------- -------------------------------------- 
+  Version                v4                                    
+  Canonical (RFC 4122)   705c6eab-a535-4f49-bd51-436d0e81206a  
+  Base 58                EsjuVs1nd42xt7jSB8hNQH                
+  Base 32                3GBHQAQ99N9X4VTMA3DM78283A            
+ ---------------------- -------------------------------------- 
+
+
+EOF
+            , $commandTester->getDisplay(true));
+    }
+
+    public function testV5()
+    {
+        $commandTester = new CommandTester(new InspectUuidCommand());
+
+        $this->assertSame(0, $commandTester->execute(['uuid' => '4ec6c3ad-de94-5f75-b5f0-ad56661a30c4']));
+        $this->assertSame(<<<EOF
+ ---------------------- -------------------------------------- 
+  Label                  Value                                 
+ ---------------------- -------------------------------------- 
+  Version                v5                                    
+  Canonical (RFC 4122)   4ec6c3ad-de94-5f75-b5f0-ad56661a30c4  
+  Base 58                AjCoyQeK6TtFemqYWV5uKZ                
+  Base 32                2ERV1TVQMMBXTVBW5DASK1MC64            
+ ---------------------- -------------------------------------- 
+
+
+EOF
+            , $commandTester->getDisplay(true));
+    }
+
+    public function testV6()
+    {
+        $commandTester = new CommandTester(new InspectUuidCommand());
+
+        $this->assertSame(0, $commandTester->execute(['uuid' => '1eb59937-b0a7-6288-a861-db3dc2d8d4db']));
+        $this->assertSame(<<<EOF
+ ---------------------- ------------------------------------------------- 
+  Label                  Value                                            
+ ---------------------- ------------------------------------------------- 
+  Version                v6                                               
+  Canonical (RFC 4122)   1eb59937-b0a7-6288-a861-db3dc2d8d4db             
+  Base 58                4nwhs6vwvNU2AbcCSD1XP8                           
+  Base 32                0YPPCKFC57CA4AGREV7Q1DHN6V                       
+ ---------------------- ------------------------------------------------- 
+  Timestamp              1610977552.4279 (2021-01-18T13:45:52.428+00:00)  
```

Comment:
@nicolas-grekas @jderusse `1610977552.4279` looks invalid to me, since the timestamp is supposed to be precise in milliseconds only, we should round somewhere isn'it?

Reply:
I just checked and it's actually OK.